Chapter4/proRing.erl

```diff
@@ -0,0 +1,85 @@
+-module(proRing).
```

Comment:
I like how you did it, I'll keep this method in mind if I have another exercise like this in the future